examples/flutter_gallery/lib/demo/material/search_demo.dart

```diff
@@ -60,7 +60,7 @@ class _SearchDemoState extends State<SearchDemo> {
                   ? Icons.more_horiz
                   : Icons.more_vert,
             ),
-            onPressed: () { },
+            onPressed: () {},
```

Comment:
Looks like you have formatting enabled, you should turn it off and undo all the formatting changes in all these files.